pkg/plugin/datasource.go

```diff
@@ -119,6 +119,20 @@ type queryModel struct {
 
 func (d *Datasource) query(ctx context.Context, query concurrent.Query) backend.DataResponse {
 	logger := log.DefaultLogger.FromContext(ctx)
+	// log panic
+	defer func() {
+		if r := recover(); r != nil {
+			var ok bool
```

Comment:
```suggestion
```

You don't need this as you `:=` the line below

Reply:
updated.